common/beerocks/bwl/shared/nl80211_socket.h

```diff
@@ -0,0 +1,80 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * Copyright (c) 2020 MaxLinear
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+#ifndef __BWL_NL80211_SOCKET_H__
+#define __BWL_NL80211_SOCKET_H__
+
+#include "nl_genl_socket.h"
+
+namespace bwl {
+
+/**
+ * @brief Extension to the generic Netlink protocol class for NL80211.
+ *
+ * The NL80211 is the 802.11 Netlink-based userspace interface for the new cfg80211 configuration
```

Comment:
```suggestion
 * The NL80211 is the 802.11 Netlink-based userspace interface for the cfg80211 configuration
```

It was introduced in v2.6.24 so it's hardly "new" any more.

Reply:
Done